flare/output.py

```diff
@@ -271,27 +265,33 @@ def write_hyps(self, hyp_labels, hyps, start_time, like, like_grad):
         :param like_grad:
         :return:
         """
-        f = self.outfiles['log']
+        f = self.outfiles[name]
         f.write('\nGP hyperparameters: \n')
 
-        for i, label in enumerate(hyp_labels):
-            f.write('Hyp{} : {} = {}\n'.format(i, label, hyps[i]))
+        if (hyp_labels):
+            for i, label in enumerate(hyp_labels):
+                f.write(f'Hyp{i} : {label} = {hyps[i]}\n')
+        else:
+            for i, hyp in enumerate(hyps):
+                f.write(f'Hyp{i} : {hyp}\n')
 
-        f.write('likelihood: ' + str(like) + '\n')
-        f.write('likelihood gradient: ' + str(like_grad) + '\n')
-        time_curr = time.time() - start_time
-        f.write('wall time from start: %.2f s \n' % time_curr)
+        f.write('likelihood: {like}\n')
+        f.write('likelihood gradient: {like_grad}\n')
+        if (start_time):
+            time_curr = time.time() - start_time
+            f.write('wall time from start: {time_curr:.2} s \n')
 
         if self.always_flush:
             f.flush()
 
     def write_gp_dft_comparison(self, curr_step, frame,
                                 start_time, dft_forces,
-                                mae, mac, local_energies=None, KE=None):
+                                mae, pmae, mac, local_energies=None, KE=None):
```

Comment:
Would suggest renaming pmae to mae_ps to mirror 'mae_perspecies"

Reply:
fixed in efd157402d391ec9091bb5041bd8a6bddf753c22